.circleci/config.yml

```diff
@@ -197,6 +204,9 @@ jobs:
           command: |
             sudo apt install openssl -y
             bash scripts/decrypt.sh
+      - run:
+          name: Generate test lists
+          command: bash scripts/generate-test-lists.sh
```

Comment:
If I understand correctly, these are being generated and then are passed to the other jobs by being saved to the workspace. For debugging purposes, can you save these as artifacts as well?
An example would be the "/tmp/dockstore-web-cache.misses.log" files we're currently saving

Reply:
I have stored the test lists.
https://app.circleci.com/pipelines/github/dockstore/dockstore/7999/workflows/df2459e2-db84-4fbd-aec8-e661a1e470b0/jobs/21241/artifacts